src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1019,7 +1019,21 @@ int RPCCall::fromCommandLine (const std::vector<std::string>& vCmd)
         }
         else
         {
-            auto const setup = setup_ServerHandler(getConfig(), std::cerr);
+            ServerHandler::Setup setup;
+            try
+            {
+                std::stringstream ss;
+                setup = setup_ServerHandler(getConfig(), ss);
+            }
+            catch(...)
+            {
+                // eat this
```

Comment:
Usually I agree, but this is one of the exceptions.

(See what I did there?)

It's done intentionally, it's noted with a (short) comment, it's very narrow in scope (parsing the config file), and the consequences are minimal (only relevant when using the command-line client).